### ISSUE TITLE
PDF to Swagger switch

### ISSUE THREAD

**Issue by SysAdminSfC** [ScienceForChange/soundCollect#27]
@CarlosMillanSfC can you check if the transition is done correctly? this will also be the way to document OdourCollect in the future.
would you be able to get this done this week? idea is to review URLs, header-body-path parameters and response bodies.

pdf: https://docs.google.com/document/d/1yjSKpc9hoHJ1c8A-Kn9naf2hQ9slXtWsDDJ1pe2CT_8/edit#heading=h.g3gv9a5rdep6
swagger: https://app.swaggerhub.com/apis-docs/science_for_change/SoundCollect/2.0.1

**Comment by CarlosMillanSfC** [comment]
Yeah, I can check it out and if i see something odd should i tell you here or create a issue with the corrections? @OtariOboladzeSfC @SysAdminSfC

**Comment by SysAdminSfC** [comment]
could you put them here as comments? like separate comment for each ones

**Comment by CarlosMillanSfC** [comment]
in the gender part i cant select nothing in the select field

**Comment by CarlosMillanSfC** [comment]
in the logout part it says:
_The API key was invalid or missing._

I think it was a type ofmyself because we are not sending any api key, i think it refers to the Bearer Token

**Comment by CarlosMillanSfC** [comment]
about the otpgenerate endpoint maybe it would be nice to add the types that are: 'verify-email' and 'new-password', they are not in the doc but i just propose

**Comment by CarlosMillanSfC** [comment]
in the post request i do not see the constraint of the data sended, i mean the optional, required, string, etc
